.github/workflows/test-spras.yml

```diff
@@ -60,10 +61,34 @@ jobs:
       - name: Run tests
         shell: bash --login {0}
         # Verbose output and disable stdout and stderr capturing
-        run: pytest -vs
+        # [this pytest-split workflow was borrowed from Snakemake CI]
+        run: pytest -vs --splits 2 --group ${{ matrix.test_group }}
```

Comment:
Cool feature. I scrolled through https://pypi.org/project/pytest-split/ to get a sense of what this does. (no action needed)